security/aqua/ansible/roles/aqua-csp/defaults/main.yml

```diff
@@ -0,0 +1,66 @@
+---
+# defaults file for aqua-csp
+# --- Aqua Common ---
+aqua_common_namespace: devops-security-aqua
+aqua_manage_common_namespace: true
+aqua_common_privileged: false
+aqua_service_account: aqua-sa
+aqua_state: present
+aqua_registry_host: registry.aquasec.com
+aqua_registry_user: "{{ lookup('env', 'AQUA_REG_USER') }}"
+aqua_registry_password: "{{ lookup('env', 'AQUA_REG_PASSWORD') }}"
+aqua_administrator_password: "{{ lookup('env', 'AQUA_ADMIN_PASSWORD') }}"
+aqua_license: "{{ lookup('env', 'AQUA_LICENSE') }}"
+aqua_use_infra_nodes: true
+
+# --- Aqua Database ---
+## Patroni Common
+aqua_patroni_suffix: ''
+aqua_patroni_cpu_request: 250m
+aqua_patroni_cpu_limit: 1000m
+aqua_patroni_memory_request: 512Mi
+aqua_patroni_memory_limit: 1024Mi
+aqua_patroni_storage_class: gluster-file
```

Comment:
I suggest putting the actual type of storage class into the inventory file and then call up the value of said variable from there. At present the PROD NetApp storage to my knowledge isn't visible in LAB, and PROD's OCS remaining storage is literally running on fumes right now (27GB free at this moment). Thus at present, we'd be using OCS storage in LAB, and NetApp storage in PROD.